section1/relection.md

```diff
@@ -1,19 +1,37 @@
+I have also found teaching to be very helpful in learning. It's hard to remember and put this into practice, but it makes a big difference.
+
+I struggle most with the taking care of your brain part, mostly because I have chronic sleep issues. And I tend to take a long time to really get into work mode, and once I do I forget to take breaks or choose not to because I don't want to lose momentum.
 
 2. What are the data types you learned about in this section? In your own words, define each.
+**String** — text, which can include letters, numbers, punctuation, and special characters. It is contained in either single or double quotes.
+**Number** — a positive or negative number. Integers are whole numbers and floats include a decimal.
+**Boolean** — either true or false. A variable can be assigned to a boolean, or a boolean can be the result of a comparison.
 
 3. How would you log the string `"Hello World!"` to the console?
+`console.log("Hello World!")`
 
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
+// two forward slashes at the beginning of a line make it a comment. Comments are to clarify what a particular chunk of code does/is for, so that it is easy to understand for someone else or future you.
 
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
+Variables are a way to store bits of data so they can be remembered and manipulated. Imagine a box that you label "shoes". You can put sandals in the box, and when you need them they'll be in the box. Any time you want, you can take the sandals out and put sneakers in the box instead. When you go looking for a pair of shoes, the sneakers will be there and not the sandals, but they are still shoes. Because they are in a box, the shoes are easier to organize, stack, etc.
 
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
+Instagram:
+userName = "string"
+passWord = "string"
+numberOfFollowers = number
 
 7. In your own words, explain what concatenation is.
+Combining variables and strings so that you can print strings with variable data inserted.
 
 8. Think of a site or app you use frequently. Where do you think the developers used concatention?
+Instagram again:
+" userName just followed you!"
 
-9. What questions do you still have about the work we've done so far? (not having a question is not an option)
\ No newline at end of file
+9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+How important is it to use a semicolon at the end of each line in Javascript? I've been doing research, and I understand that it is "optional" because JS will insert it if you don't, but is it the sort of thing where it is considered more professional/better form to always include them?
```

Comment:
You'll find that there are "conventions" like this that vary from team to team and you'll just need to be flexible and go with whatever convention your organization or team has agreed upon. Another example is how branches are named or how commit messages or formulated. Fortunately things like semicolons and other syntax are easy for a linter to enforce and correct so you should have to worry about it too much in the real world!